rosdep/python.yaml

```diff
@@ -882,6 +882,7 @@ python-catkin-pkg:
   ubuntu:
     artful: [python-catkin-pkg]
     bionic: [python-catkin-pkg]
+    focal: [python3-catkin-pkg]
```

Comment:
This is a Python 3 package, but it is a Python 2 key. From what I can tell, there is no Python 2 package for `catkin_pkg` in Focal.

Reply:
@up201305618 if there's no python 2 version of the package for focal, it cannot be added.
See the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md).